From 48f092ae435bb9fe5087b92d4a990a0184a2608e Mon Sep 17 00:00:00 2001
From: Adam Duskett <Aduskett@gmail.com>
Date: Tue, 8 Dec 2020 17:55:05 -0800
Subject: [PATCH] main.conf patch

Signed-off-by: Adam Duskett <Aduskett@gmail.com>
---
 src/main.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.conf b/src/main.conf
index 16701eb..dd8bf95 100644
--- a/src/main.conf
+++ b/src/main.conf
@@ -126,4 +126,4 @@
 # AutoEnable defines option to enable all controllers when they are found.
 # This includes adapters present on start as well as adapters that are plugged
 # in later on. Defaults to 'false'.
-#AutoEnable=false
+AutoEnable=true
-- 
2.29.2

